app/controllers/games_controller.rb

```diff
@@ -1,17 +1,19 @@
 class GamesController < ApplicationController
+  include GamesHelper
```

Comment:
Rails should do this. Unless you want to use the helper methods in the controller, in which case: DONT. Write the methods here. Helpers are for the views